### ISSUE TITLE
E722 (bare except) not reported.

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#611]
In GitLab by @aidalgol on Aug 4, 2017, 14:23

Flake8 installed via `pip install flake8` inside a Python2.7 virtual environment.

Output of `flake8 --bug-report`:
```
{
  "dependencies": [
    {
      "dependency": "setuptools", 
      "version": "36.2.7"
    }
  ], 
  "platform": {
    "python_implementation": "CPython", 
    "python_version": "2.7.13", 
    "system": "Linux"
  }, 
  "plugins": [
    {
      "plugin": "mccabe", 
      "version": "0.6.1"
    }, 
    {
      "plugin": "naming", 
      "version": "0.4.1"
    }, 
    {
      "plugin": "pycodestyle", 
      "version": "2.3.1"
    }, 
    {
      "plugin": "pyflakes", 
      "version": "1.5.0"
    }
  ], 
  "version": "3.4.1"
}
```

When `flake8` is run without arguments, and with no configuration file present, on the following file...
```python
#!/usr/bin/env python2

if __name__ == "__main__":
    try:
        print "Never raises exception"
    except:
        print "Bare exception handler"
```
...no errors, warnings, or violations are reported, even though `pycodestyle` reports the bare exception handler:
```
bare-except.py:6:5: E722 do not use bare except'
```

**Comment by asottile** [comment]
In GitLab by @aidalgol on Aug 4, 2017, 14:24

changed title from **E722 (bare {-`except`-}) not reported.** to **E722 (bare {+except+}) not reported.**

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Aug 5, 2017, 05:27

Nice catch! I have a fix for this ready for 3.5.0 since it enables some new checks and I don't want those turning on for folks in a patch release.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Aug 5, 2017, 05:28

mentioned in merge request !199

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Aug 6, 2017, 12:09

closed via merge request !199

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Aug 6, 2017, 12:09

closed via commit 6e656c64c79146e79775a89a70fdf8249014e2bb

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Aug 6, 2017, 12:09

closed via commit ee435593488a5c3b9805bc686b83d17505491c42

**Comment by asottile** [comment]
In GitLab by @unofficial-mirror on Nov 21, 2017, 19:34

mentioned in commit unofficial-mirrors/ceph@3600cd7bfdca76485d2998c5da1a0dd25816d1ce

**Comment by asottile** [comment]
In GitLab by @unofficial-mirror on Nov 26, 2017, 09:26

mentioned in commit unofficial-mirrors/ceph@47ab677d907267930aedfcd76fc7b8c7b2610bd4

**Comment by asottile** [comment]
In GitLab by @unofficial-mirror on Nov 28, 2017, 05:37

mentioned in commit unofficial-mirrors/ceph@bb03f6b20b4963ad5d51c1c6f45f19bc7eaf220b
